doc/diagrams/c4-model/views.dsl

```diff
@@ -35,6 +35,13 @@ workspace extends "https://raw.githubusercontent.com/Energinet-DataHub/opengeh-a
             include ->certificatesDomain->
             autoLayout
         }
+        container energyOrigin "ContainerContextEO" {
+            title "[Container Context] Energy Origin"
+            include *
+            autoLayout
+        }
+
+        # Specific area container views
```

Comment:
Suggest moving the view for Certificates down here

Reply:
Will do!